application/redis-mysql-helidon/src/test/java/webapp/tier/service/MysqlServiceTest.java

```diff
@@ -0,0 +1,14 @@
+package webapp.tier.service;
+
+import org.junit.jupiter.api.Test;
+
+class MysqlServiceTest {
+
+	@Test
+	void test() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5870129)

Reply:
f0a70ca